src/setup.rs

```diff
@@ -94,3 +97,25 @@ pub fn check_secret_env_var() {
         },
     }
 }
+
+pub fn configure_cors() -> CorsLayer {
+    let result = std::env::var("FRONTEND_ORIGIN");
+
```

Comment:
Suggested change would make this only run in release builds.
```suggestion
    #[cfg(not(debug_assertions))]
    if result.is_err() {
        error!("{}", FRONTEND_ORIGIN_NOT_SET);
        panic!();
    }
```